lib/components/split-pane.js

```diff
@@ -76,28 +76,47 @@ export default class SplitPane extends Component {
 
   template(css) {
     const children = this.props.children;
-    const {direction, borderColor} = this.props;
+    const {direction, borderColor, backgroundColor} = this.props;
     let {sizes} = this.props;
     if (!sizes) {
       // workaround for the fact that if we don't specify
       // sizes, sometimes flex fails to calculate the
       // right height for the horizontal panes
       sizes = new Array(children.length).fill(1 / children.length);
     }
+
     return (<div className={css('panes', `panes_${direction}`)}>
       {
         React.Children.map(children, (child, i) => {
+          const {maximized} = child.props.termGroup;
           const style = {
             flexBasis: (sizes[i] * 100) + '%',
             flexGrow: 0
           };
+          const maximizedStyle = {
+            position: 'absolute',
+            top: 0,
+            right: 0,
+            bottom: 0,
+            left: 0,
+            zIndex: 1000,
+            backgroundColor
+          };
           return [
             <div
               key="pane"
               className={css('pane')}
-              style={style}
+              style={maximized ? maximizedStyle : style}
               >
               { child }
+              { maximized &&
+                <div className={css('maximized_icon')}>
+                  <div
+                    className={css('maximized_icon_inner_dot')}
+                    style={{backgroundColor}}
+                    />
+                </div>
+              }
```

Comment:
This icon indicates if pane is maximized and there're more panes are hidden (see gif above).